cs/datatypes/ref.adoc

```diff
@@ -0,0 +1,222 @@
+= Datový typ `ref!`
+:toc:
+:numbered: 
+
+== Úvodem
+
+Hodnotami typu `ref!` jsou pojmenované odkazy, jako jsou názvy účtů u sociálních medií nebo identifikátory s implikovaným názvem domény.
+
+Datový typ `ref!` patři do typesetu `any-string!`, přičemž je konceptuálně blízký datovému typu `issue!`, od něhož se liší v implementaci (string-like vs. symbol-like) a v poskytované sadě kompromisů.
```

Comment:
"String-like vs. symbol-like". @Tovim do you need any help in translating technical passages like this one? I authored [an explanation](https://github.com/red/red/wiki/%5BDOC%5D-issue!-vs.-ref!) on the subject, maybe it will be of some help (although it's written in English).